pkg/common/naming.go

```diff
@@ -90,6 +90,10 @@ func PVCCleanupJobName(workspaceId string) string {
 	return fmt.Sprintf("cleanup-%s", workspaceId)
 }
 
+func PerWorkspacePVCName(workspaceId string) string {
+	return fmt.Sprintf("storage-%s", workspaceId)
```

Comment:
maybe let's call it claim?

Reply:
would be more consistent IMO:
- claim-che-workspace (old PVCs)
- claim-devworkspace (new common PVC)
- claim-workspaceid (new per-workspace PVC)

Reply:
We're flip-flopping on our suggestions :) 

https://github.com/devfile/devworkspace-operator/pull/818#discussion_r849648541

Reply:
I ended up going with `storage` for now since it seems to be a bit more representative of what it's purpose is, though I'm not going to make a fuss :P 

Should be good to merge after finally CI run.